src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -138,13 +138,13 @@ export const monoPlatform: Platform = {
     return ((baseAddress as any as number) + (fieldOffset || 0)) as any as T;
   },
 
-  beginHeapLock: function() {
+  beginHeapLock: function () {
```

Comment:
Sorry about this - I formatted the file and it made a bunch of changes. Use the Hide whitespaces option to avoid seeing unnecessary changes.

Reply:
No problem. Best not to fight the tools :) If this is how VS code wants to format it, let's do it like that!